### PR TITLE
RFC for zkVM CLI/Host integration

### PR DESCRIPTION
**Pull request by rbonichon**

### CODE REVIEW

File: 0011-cli-host-integration.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+### Response format
+
+The response, of variable length, consists of a byte stream with a
+8-bytes (`uint64`) length prefix together with `length`-bytes pre-image
+data.
+
+``` {#preimage_response_format .text}
++------------+--------------------+
+| length <8> | pre-image <length> |
```
Comment (fabrizio-m) [inline 0011-cli-host-integration.md:111]: Is here `pre-image` the actual data? or the address in memory where it was mapped?

File: 0011-cli-host-integration.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+---
```
Comment (mrmr1993) [inline 0011-cli-host-integration.md:1]: This doesn't appear to be a markdown document.

**Review by mrmr1993 — COMMENTED** [review]
Could benefit from more detail, but what's here LGTM so far.

File: 0011-cli-host-integration.md (hunk 1 above)
Comment (rbonichon) [reply 0011-cli-host-integration.md:111]: It is the data indeed.

**Comment by joseandro** [comment]
Closing this as we will defer to using our code documentation mirroring 1:1 the OP stack. cc @rbonichon @mrmr1993

**Pull request closed without merging**